plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -1382,29 +1382,29 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase {
   }
 
   internal fun handleSingleTapUpEvent(): Boolean {
-    cameraAnimationsPlugin.cancelAllAnimators(protectedCameraAnimatorOwnerList)
+    cameraAnimationsPlugin.cancelAllAnimators(protectedCameraAnimatorOwners)
     return true
   }
 
   private fun notifyOnFlingListeners() {
-    if (!onFlingListenerList.isEmpty()) {
-      for (listener in onFlingListenerList) {
+    if (!onFlingListeners.isEmpty()) {
```

Comment:
do really need this check and all below?

Reply:
Removed these isEmpty checks